### PR TITLE
Add rlcraft example

### PR DESCRIPTION
**Pull request by Midnight-Conqueror**
After quite a bit of trial and error I finally figured out how to get rlcraft running using docker compose. Here is that example

### CODE REVIEW

File: examples/docker-compose-rlcraft.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+services:
+    mc:
+      image: itzg/minecraft-server:1.9.0
+      volumes:
+        - /modpacks:/modpacks
```
Comment (DanielRamp) [inline examples/docker-compose-rlcraft.yml:7]: I think the modpacks volume (``/modpacks:/modpacks``) contains a typo. Probalby one of eighter is meant:
a) relative path: ``./modpacks:/modpacks``
b) docker-volume: ``modpacks:/modpacks``

Comment (itzg) [reply examples/docker-compose-rlcraft.yml:7]: @Midnight-Conqueror I agree wit @DanielRamp . Can you please adjust this?

File: examples/docker-compose-rlcraft.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+services:
+    mc:
+      image: itzg/minecraft-server:1.9.0
```
Comment (itzg) [inline examples/docker-compose-rlcraft.yml:5]: I would prefer not to pin this to a specific version. Instead reference one of the tags.

**Review by itzg — CHANGES_REQUESTED** [review]

**Comment by itzg** [comment]
Superseded by #1036

**Pull request closed without merging**